include/bluetooth/mesh/main.h

```diff
@@ -128,6 +128,15 @@ struct bt_mesh_prov {
 	 */
 	int         (*input)(bt_mesh_input_action_t act, u8_t size);
 
+	/** @brief Unprovisioned beacon has been received.
+	 *
+	 * This callback notifies the application that an unprovisioned
+	 * beacon has been received with the given UUID.
+	 *
+	 * @param uuid UUID
+	 */
+	void        (*unprovisioned_beacon)(u8_t uuid[16]);
```

Comment:
Could you expose the URI hash here as well?

Reply:
Yes and probably the OOB Information as well?

Reply:
Oh yeah, good point!

Reply:
Added OOB info and URI hash.